sbi/utils/progress.py

```diff
@@ -0,0 +1,13 @@
+class no_tqdm(object):
```

Comment:
Do we need this class? Am I missing something, or could we just use `disable` kwarg of `pbar`?

Reply:
Your're right, this worked! Thanks :)